package.json

```diff
@@ -35,6 +35,7 @@
   },
   "devDependencies": {
     "@babel/core": "7.18.2",
+    "@chakra-ui/icons": "2.0.2",
```

Comment:
Currently we don't have our own icon library (WIP), I'll just start with `@chakra-ui/icons` for visual testing on StoryBook

Reply:
The designers started already working on the figma-icon-library. You can find it here -> https://www.figma.com/file/K67jAk1bv1qmnmPVsf2we4/AS24-Icons?node-id=96%3A817
There you will find the icon you need with the correct size.

Please add the corresponding icon instead using `@chakra-ui/icons`. Thank you!

Reply:
I think when creating the Component, we should add enough flexibility to its usage and the component itself should not be depended on any page/forms. If you have a look on the [WebComponent figma](https://www.figma.com/file/CHU4fpMfXpiAW4qNFzi9y1/AS24-WebComponents?node-id=359%3A15807), you'll see it's not only the `arrow-left` icon, but also unlimited chances to display other icons

Btw, I created a [new discussion](https://github.com/smg-automotive/au-docs/discussions/16) for this topic. We can keep discussing further there